Utils/Dataflow/pyDKB/dataflow/stage/AbstractStage.py

```diff
@@ -54,6 +89,31 @@ def parse_args(self, args):
             raise ValueError(
                 "Parameter -m|--mode must be used with value: -m MODE.")
 
+        if self.ARGS.eom is None:
+            self.ARGS.eom = '\n'
+        elif self.ARGS.eom == '':
+            raise ValueError("(ERROR) Empty EOM marker specified.\n")
+        else:
+            try:
+                self.ARGS.eom = self.ARGS.eom.decode('string_escape')
+            except (ValueError), err:
+                sys.stderr.write("(ERROR) Failed to read arguments.\n"
+                                 "Case: %s\n" % (err))
```

Comment:
For multiline log messages every line should be prepended with `(ERROR)` (or only the first line, and all the others with `(==)` [1].

It is required due to the implementation of the [ExternalProcessLogger](https://github.com/PanDAWMS/dkb/blob/master/Utils/Dataflow/000_kafka/ru/kiae/dkb/kafka/common/external/ExternalProcessLogger.java). Basically, it does the following: it looks for `(ERROR|WARN|WARNING|INFO|DEBUG|INFO)` marker at the beginning of the line read from the subprocess\` STDERR, and writes the line to the corresponding log level of the Java program. So when the Java program runs this or that DKB stage script, it has log messages from the stage in its own log file, and the one reading this log can check what happened during the data processing on every stage.

However when there\`s no log level marker, ExternalProcessLogger takes `(TRACE)` level as default -- and if logging level is set to something else (say, `(INFO)`), this line, even being a second line of the `(ERROR)` message, won`t appear in the log file and important information will be missed.

---
[1] The part of `(==)` is not implemented yet on the Java part, yet one day it will be and will work as "same as above".